testsuite/features/support/constants.rb

```diff
@@ -72,12 +72,14 @@
   'salt_migration_minion' => 'SALT_MIGRATION_MINION'
 }.freeze
 
-ADDRESSES = {
+# TODO: the values for pxeboot_minion, sle12sp5_terminal, sle15sp4_terminal, and proxy can now be set in sumaform
+#       remove them from this array when we read them from .bashrc
+PRIVATE_ADDRESSES = {
   'network'           => '0',
-  'sle_minion'        => '3',
```

Comment:
Aside of retail, I wonder if we don't use this value for sle_minion in another scope. 👀

Reply:
sle minion does not even have a networking card in the private network anymore. So it is strictly impossible.

And also, I checked with a grep ;-) . And the tests passed.

Reply:
Cool.